src/insns/cjalr_jalr_32bit.adoc

```diff
@@ -24,11 +24,20 @@ Capability Mode Description::
 CJALR allows unconditional, indirect jumps to a target capability. The
 target capability is obtained by unsealing `cs1` and incrementing its address by the
 sign-extended 12-bit immediate, and then setting the least-significant bit of the
-result to zero. The target capability may have
+result to zero. The target address may then have
 xref:section_invalid_addr_conv[xrefstyle=short]
-performed and is then installed in <<pcc>>. The <<pcc>>
-of the next instruction following the jump (<<pcc>> + 4) is sealed and written
-to `cd`.
+performed. The check for length violation is done next using the converted
+address and the original bounds.
+
+[NOTE]
+====
+When performing invalid address conversion the tag never needs to be cleared because
+the earlier length violation check is a stricter condition than the representability
+check, and the capability is always unsealed before invalid address conversion.
+====
+
+The capability is then installed in <<pcc>>. The <<pcc>> of the next instruction
+following the jump (<<pcc>> + 4) is sealed and written to `cd`.
```

Comment:
Please note that that this test is wrapped under the `Synopsis::` header, but it is not currently indented properly because it is after the note and it is also missing the + at the end of the previous paragraph.